1.12/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# vim:set ft=dockerfile:
+
+FROM cimg/base:2021.05
```

Comment:
Can you down-rev this base image version to 2021.04?  We tend to only update the base version occasionally (and in unison across our convenience images) in order to ensure that these base images are cached properly.